portal/src/main/webapp/js/src/oncoprint/webgl/dist/oncoprint-bundle.js

```diff
@@ -1656,14 +1657,17 @@ var OncoprintLabelView = (function () {
 		} else {
 		    var hovered_track = isMouseOnLabel(view, evt.pageY - view.$canvas.offset().top);
 		    if (hovered_track !== null) {
-			var $tooltip_div = $('<div>');
+			var $tooltip_div = $('<section>');
```

Comment:
Ah, no, sorry, I should probably have referred to some documentation. I believe they were introduced in HTML5, along with the `<nav>` tag, to signal additional semantic information to future browsers and alternative viewers (such as screen readers for visually impaired people). Here's one document about it: https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Using_HTML_sections_and_outlines